packages/lit-dev-content/src/components/litdev-tutorial.ts

```diff
@@ -774,20 +789,17 @@ export class LitDevTutorial extends LitElement {
     };
   }
 
-  private reportMetrics(idx: number, tutorialUrl: string) {
-    if (idx === 0 || !hasRecordedStart) {
-      window.gtag?.('event', 'tutorial_start', {
-        category: 'tutorials',
-        event_label: tutorialUrl,
-        value: idx,
-      });
-      hasRecordedStart = true;
-    } else if (idx === this._manifest.steps.length - 1 && !hasRecordedEnd) {
-      window.gtag?.('event', 'tutorial_end', {
-        category: 'tutorials',
-        event_label: tutorialUrl,
-      });
-      hasRecordedEnd = true;
+  private _handleTutorialMetricEvent(event: TutorialMetricEvent) {
+    switch (event) {
+      case 'tutorial_start':
+        hasRecordedStart = true;
+        break;
+      case 'tutorial_end':
+        hasRecordedEnd = true;
+        break;
+      case 'tutorial_progress':
+      default:
```

Comment:
A `never` check would be good. Otherwise the `default` case doesn't need to be here.